### DISCUSSION TITLE
How to trigger cross repo workflows with Dependency

### DISCUSSION THREAD

**Discussion by Kirankumar-ceq** [community/community#28192] [Actions]
I wanted to trigger workflows using cross repos. suppose  I have two repos in one org repo A and Repo B. after the successful completion of workflow A, B will be triggered automatically

**Comment by airtower-luna** [comment]
There's nothing built-in for triggers across repositories.

What you could do is have in repository A [send a `workflow_dispatch` event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) at the end of its workflow, or (maybe better) in a separate workflow triggered with [`workflow_run`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run), to the workflow you want to run in repository B. The catch is that doing that requires a PAT with appropriate permissions.
